examples/static-hbase-encrypted/networking.tf

```diff
@@ -0,0 +1,75 @@
+module "sg-ports" {
+  # source               = "git::https://github.com/Datatamer/terraform-aws-emr.git//modules/aws-emr-ports?ref=7.3.0"
+  source       = "../../modules/aws-emr-ports"
+  applications = local.this_application
+}
+
+module "aws-emr-sg-master" {
+  source                  = "git::git@github.com:Datatamer/terraform-aws-security-groups.git?ref=1.0.0"
+  vpc_id                  = var.vpc_id
+  ingress_cidr_blocks     = var.ingress_cidr_blocks
+  ingress_security_groups = []
+  egress_cidr_blocks      = var.egress_cidr_blocks
+  ingress_ports           = module.sg-ports.ingress_master_ports
+  sg_name_prefix          = format("%s-%s", var.name_prefix, "-master")
+  egress_protocol         = "all"
+  ingress_protocol        = "tcp"
+}
+
+module "aws-emr-sg-core" {
+  source                  = "git::git@github.com:Datatamer/terraform-aws-security-groups.git?ref=1.0.0"
+  vpc_id                  = var.vpc_id
+  ingress_cidr_blocks     = var.ingress_cidr_blocks
+  ingress_security_groups = []
+  egress_cidr_blocks      = var.egress_cidr_blocks
+  ingress_ports           = module.sg-ports.ingress_core_ports
+  sg_name_prefix          = format("%s-%s", var.name_prefix, "-core")
+  egress_protocol         = "all"
+  ingress_protocol        = "tcp"
+}
+
+module "aws-emr-sg-service-access" {
+  source              = "git::git@github.com:Datatamer/terraform-aws-security-groups.git?ref=1.0.0"
+  vpc_id              = var.vpc_id
+  ingress_cidr_blocks = var.ingress_cidr_blocks
+  egress_cidr_blocks  = var.egress_cidr_blocks
+  ingress_ports       = module.sg-ports.ingress_service_access_ports
+  sg_name_prefix      = format("%s-%s", var.name_prefix, "-service-access")
+  egress_protocol     = "all"
+  ingress_protocol    = "tcp"
+}
+
+# Creates KMS VPC Endpoint
+module "endpoints" {
+  source = "terraform-aws-modules/vpc/aws//modules/vpc-endpoints"
```

Comment:
We could add this to the networking module in the future. What do you think @souza-dan ?

Reply:
Agreed, this would be good to add to the networking module as a follow-on.

Reply:
Ticket [CA-186](https://datatamr.atlassian.net/browse/CA-186) was opened for this.
Thank you,